proposed/http-handlers/request-handlers.md

```diff
@@ -110,20 +110,20 @@ use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
 
 /**
+ * Participant in processing a server request and response
+ *
  * An HTTP middleware component participates in processing an HTTP message,
- * either by acting on the request or the response. This interface defines the
- * methods required to use the middleware.
+ * either by acting on the request, generating the response, or forwarding the
+ * request to a subsequent middleware and possibly acting on its response.
```

Comment:
s/either by/for example by/ ?

That avoids any potential misinterpretation as a limitation for implementations. For example some implementations would not act on the request or response at all, they would just do side-effects (e.g. a "logger middleware"). It *may* be covered by what's already described above but why take the risk of confusion.

The strict definition is the method signature (which can be summed up as `Processes a server request and produces a response`), the rest of the docblock is just to help understand, not limit the implementations.